alertlogic/service.py

```diff
@@ -18,6 +18,18 @@ def set_session(self, session):
         """
         self._session = session
 
+    def get(self, path_parts, params=None, json=None):
+        return self.call_endpoint('GET', path_parts, params, json)
+
+    def post(self, path_parts, params=None, json=None):
```

Comment:
cosmetic: i would call params -> "query_params", as it is a bit confusing which params i whould pass here. Could be client timeout, or other http request related things.

Reply:
I agree query_params it is